src/v1beta1/budget_service_client.ts

```diff
@@ -752,15 +752,29 @@ export class BudgetServiceClient {
   /**
    * Return a fully-qualified budget resource name string.
    *
+   * @param {string} billing_account
    * @param {string} budget
    * @returns {string} Resource name string.
    */
-  budgetPath(budget: string) {
+  budgetPath(billingAccount: string, budget: string) {
     return this._pathTemplates.budgetPathTemplate.render({
+      billing_account: billingAccount,
       budget,
     });
   }
 
+  /**
+   * Parse the billing_account from Budget resource.
+   *
+   * @param {string} budgetName
+   *   A fully-qualified path representing Budget resource.
+   * @returns {string} A string representing the billing_account.
+   */
+  matchBilling_accountFromBudgetName(budgetName: string) {
```

Comment:
This name is wrong, must be `matchBillingAccountFromBudgetName`. 

@bcoe Please do not release this library, we'll fix it soon.

@xiaozhenliu-gg5 Can you fix it in scope of your resource names PR?

Thank you!

Reply:
Sure, this should be fixed by PR: https://github.com/googleapis/gapic-generator-typescript/pull/213 Thanks